hooks/command

```diff
@@ -29,6 +29,11 @@ readonly log_complete_timeout_sec="${BUILDKITE_PLUGIN_K8S_LOG_COMPLETE_TIMEOUT_S
 readonly use_agent_node_affinity=${BUILDKITE_PLUGIN_K8S_USE_AGENT_NODE_AFFINITY:-false}
 readonly print_resulting_job_spec=${BUILDKITE_PLUGIN_K8S_PRINT_RESULTING_JOB_SPEC:-false}
 readonly jobs_cleanup_via_plugin=${BUILDKITE_PLUGIN_K8S_JOBS_CLEANUP_VIA_PLUGIN:-}
+readonly skip_on_macos=${BUILDKITE_PLUGIN_K8S_SKIP_ON_MACOS:-}
+
+if [[ "$skip_on_macos" == "true" && $OSTYPE == darwin* ]]; then
+  exec /bin/bash -c "$BUILDKITE_COMMAND"
```

Comment:
A command isn't required if a step uses a plugin, we should check and exec only if `BUILDKITE_COMMAND` isn't empty

Reply:
and what are you thinking we do in the other case? my thinking is that if users override those other vars that the normal logic uses, it won't make sense to use those _without_ the plugin?

Reply:
I see passing an empty string to 'bash -c' is a noop so this should be fine in any case!